examples/camera-app/linux/include/pushav-clip-recorder.h

```diff
@@ -76,23 +76,23 @@ class PushAVClipRecorder
-        uint16_t mInitialDurationS;                           ///< Initial clip duration in seconds
-        uint16_t mAugmentationDurationS;                      ///< Duration increment on motion detect
-        uint16_t mChunkDurationMs;                            ///< Chunk duration  milliseconds
-        uint16_t mSegmentDurationMs;                          ///< Segment duration in milliseconds
-        uint16_t mBlindDurationS;                             ///< Duration without recording after motion stop
-        uint16_t mPreRollLengthMs;                            ///< Pre-roll length in milliseconds
-        uint16_t mElapsedTimeS;                               ///< Elapsed time since recording start in seconds
-        std::string mOutputPath;                              ///< Base output directory path
-        std::string mTrackName;                               ///< Track name for segmented files
-        std::string mUrl;                                     ///< URL for uploading clips;
-        int mTriggerType;                                     ///< Recording trigger type
-        std::chrono::steady_clock::time_point activationTime; ///< Time when the recording started
+        bool mHasVideo;                                        ///< Video recording enabled flag
+        bool mHasAudio;                                        ///< Audio recording enabled flag
+        uint64_t mSessionNumber;                               ///< Session number for unique clip identification
+        uint8_t mSessionGroup;                                 ///< Session group for grouping multiple transports
+        uint32_t mMaxClipDurationS;                            ///< Maximum clip duration in seconds
+        uint16_t mInitialDurationS;                            ///< Initial clip duration in seconds
+        uint16_t mAugmentationDurationS;                       ///< Duration increment on motion detect
+        uint16_t mChunkDurationMs;                             ///< Chunk duration  milliseconds
+        uint16_t mSegmentDurationMs;                           ///< Segment duration in milliseconds
+        uint16_t mBlindDurationS;                              ///< Duration without recording after motion stop
+        uint16_t mPreRollLengthMs;                             ///< Pre-roll length in milliseconds
+        uint16_t mElapsedTimeS;                                ///< Elapsed time since recording start in seconds
+        std::string mOutputPath;                               ///< Base output directory path
+        std::string mTrackName;                                ///< Track name for segmented files
+        std::string mUrl;                                      ///< URL for uploading clips;
+        int mTriggerType;                                      ///< Recording trigger type
+        std::chrono::steady_clock::time_point activationTimeS; ///< Time when the recording started
```

Comment:
This is already well-defined type for "instants" of time, and shouldn't be accompanied by units. Infact, it doesn't store the value in seconds at all
```suggestion
        std::chrono::steady_clock::time_point activationTime; ///< Time when the recording started
```

We should only need `Ms` or `S` units for other raw data types (like uint16 and such) that you already have.

Reply:
Noted, updated changes.